src/libs/antares/study/binding_constraint/BindingConstraint.h

```diff
@@ -666,13 +692,21 @@ class BindingConstraintsList final : public Yuni::NonCopyable<BindingConstraints
     */
     void markAsModified() const;
 
+    [[nodiscard]] const std::map<std::string, BindingConstraintTimeSeriesNumbers>& TimeSeries() const {
+        return time_series;
+    }
+    void resizeAllTimeseriesNumbers(unsigned nb_years);
+
 private:
     bool internalSaveToFolder(BindingConstraint::EnvForSaving& env) const;
 
 private:
     //! All constraints
     BindingConstraint::Vector pList;
 
+public:
+    std::map<std::string, BindingConstraintTimeSeriesNumbers> time_series;
```

Comment:
Renaming : 
**time_series** : should be rename into **groupToTSnumbers** (for example)